action.yml

```diff
@@ -74,6 +78,7 @@ runs:
         push: ${{ inputs.push }}
         tags: ${{ steps.meta.outputs.tags }}
         labels: ${{ steps.meta.outputs.labels }}
+        platforms: ${{inputs.platforms}}
```

Comment:
Should we distribute the builds for different platforms?
https://docs.docker.com/build/ci/github-actions/multi-platform/

Reply:
This would use multiple runners (effectively run parallel insteaed of in series) resulting in lower build time.

Reply:
The caller of this action can setup parallel runners if they desire.